.github/workflows/deploy-canary.yml

```diff
@@ -0,0 +1,31 @@
+name: Canary Release
+
+on:
+  push:
+    branches:
+      - master
+
+
+jobs:
+  test:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Node (10)
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+        registry-url: 'https://registry.npmjs.org/'
+    - name: Yarn install
+      run: yarn
+    - name: yarn build
+      run: yarn build
+    - name: Deploy canary
+      run: |
+        npm config set //registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN
```

Comment:
Doesn't this need to go through wombot?

Reply:
It should eventually but the Travis config did not use it. Trying to change the minimum number of things at first in the move.